### DISCUSSION TITLE
How to get detailed info of a port

### DISCUSSION THREAD

**Discussion by Leonezz** [serialport/serialport-rs#210] [Q&A]
Currently avaliable_ports shows all my ports are of type Unknown, and the port name is a simple COM1/COM2...
But the windows divice mgr shows:
![image](https://github.com/user-attachments/assets/7e56435a-10a8-4398-a99b-fac6dbf09f1b)
The question is: is it possible to get the types correct and names complete? and how if possible?

**Comment by sirhcel** [comment]
Hello @Leonezz, to be honest, I have no idea where this information is provided from? Do you know where the Device Manager collects this information from? What does the `list_ports` example report for this port?
